.ci/ppc64le/kubernetes/init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+network_plugin_config_file="${SCRIPT_PATH}/../../.ci/${arch}/kubernetes/kube-flannel.yml"
+
+curl -fsL $flannel_url -o $network_plugin_config_file
```

Comment:
fyi / aside / if it makes a difference: we're using [`v0.13.0-rc2`](https://github.com/kata-containers/tests/blob/main/versions.yaml#L52) but the latest version is [`v.0.14.0`](https://github.com/flannel-io/flannel/releases).